enterprise/frontend/src/embedding-sdk/jest/setup-env.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line no-undef
+process.env.IS_EMBEDDING_SDK = true;
```

Comment:
This makes sure `isEmbeddingSdk` in SDK unit tests is set to `true` like in the real environment.

https://github.com/metabase/metabase/blob/af55313e5de15a4d9e1f0e665a02571eb57813c2/frontend/src/metabase/env.ts#L42-L45